en/performance/sizing-feeding.html

```diff
@@ -150,15 +138,36 @@ <h3 id="debugging-performance">Debugging performance</h3>
 -->
 
 
+<h2 id="index">Index</h2>
+<p>
+  Changes to index fields are written to the <a href="#document-store">document store</a>
+  and the <a href="../proton.html#index">index</a>. See <em>Index Manager + Index</em> in illustration above.
+</p>
+<pre>
+schema music {
+    document music {
+        field artist type string {
+            indexing: summary | index
+        }
+</pre>
+<p>
+  Both PUTs and UPDATEs to the index are memory only operations, but use CPU to update the index.
```

Comment:
Update might not be a memory only operation, since it might trigger a read from document store and a later write to document store.